notebooks/Solutions/Causal Interpretation for Ames Housing Price.ipynb

```diff
@@ -0,0 +1,3055 @@
+      "text/plain": [
+       "<Figure size 576x684 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# plot the feature importance\n",
+    "shap.summary_plot(shap_values, x_test_ohe)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "From the summary plot above, we could see the **most important features** sorted by their importance level. It tells us that houses that measure high in overall quality, square footage, and are also relatively new will have a higher housing price. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Causal Interpretation\n",
+    "### Direct Causal Effect --  Do certain house amenities have a direct effect on home value?"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "<econml.solutions.causal_analysis._causal_analysis.CausalAnalysis at 0x1c083a14610>"
+      ]
+     },
+     "execution_count": 15,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from econml.solutions.causal_analysis import CausalAnalysis\n",
+    "\n",
+    "# initialize heterogeneity features\n",
+    "hetero_cols = ['AgeAtSale', 'MSZoning']\n",
+    "ca = CausalAnalysis(\n",
+    "    feature_inds= [\n",
+    "        'HasFireplace',\n",
+    "        'HasPorch',\n",
+    "        'HasDeck',\n",
+    "    ],\n",
```

Comment:
Why these three features?  Overall, this new notebook is good, but one thing that's missing compared to the old one is any finding that one of top predictive features may not be causally important.

Maybe the dataset doesn't support such a finding, and that would be fine to point out too (something like "We find that not only are features X, Y, and Z predictive of high prices, but that even when accounting for other features they appear to be causally linked.")  But right now there doesn't seem to be a connection between the predictive and causal analysis.